cmd/api/main.go

```diff
@@ -187,28 +179,39 @@ func main() {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 	mux := runtime.NewServeMux()
-	err = v1alpha2pb.RegisterResultsHandlerFromEndpoint(ctx, mux, ":"+serverConfig.GRPC_PORT, opts)
+	err = v1alpha2pb.RegisterResultsHandlerFromEndpoint(ctx, mux, ":"+serverConfig.SERVER_PORT, opts)
 	if err != nil {
 		log.Fatal("Error registering gRPC server endpoint: ", err)
 	}
 
 	if serverConfig.LOGS_API {
-		err = v1alpha2pb.RegisterLogsHandlerFromEndpoint(ctx, mux, ":"+serverConfig.GRPC_PORT, opts)
+		err = v1alpha2pb.RegisterLogsHandlerFromEndpoint(ctx, mux, ":"+serverConfig.SERVER_PORT, opts)
 		if err != nil {
 			log.Fatal("Error registering gRPC server endpoints for log: ", err)
 		}
 	}
 
-	// Start REST proxy server
-	log.Infof("REST server Listening on: %s", serverConfig.REST_PORT)
+	// Start server with gRPC and REST handler
+	log.Infof("gRPC and REST server listening on: %s", serverConfig.SERVER_PORT)
 	if tlsError != nil {
-		log.Fatal(http.ListenAndServe(":"+serverConfig.REST_PORT, mux))
+		log.Fatal(http.ListenAndServe(":"+serverConfig.SERVER_PORT, grpcHandlerFunc(gs, mux)))
 	} else {
-		log.Fatal(http.ListenAndServeTLS(":"+serverConfig.REST_PORT, path.Join(serverConfig.TLS_PATH, "tls.crt"), path.Join(serverConfig.TLS_PATH, "tls.key"), mux))
+		log.Fatal(http.ListenAndServeTLS(":"+serverConfig.SERVER_PORT, path.Join(serverConfig.TLS_PATH, "tls.crt"), path.Join(serverConfig.TLS_PATH, "tls.key"), grpcHandlerFunc(gs, mux)))
 	}
 
 }
 
+// grpcHandlerFunc forwards the request to gRPC server based on the Content-Type header.
+func grpcHandlerFunc(grpcServer *grpc.Server, otherHandler http.Handler) http.Handler {
```

Comment:
nit - refer to the other as "rest" or "http"
```suggestion
func grpcHandlerFunc(grpcServer *grpc.Server, httpServer http.Handler) http.Handler {
```